hooks/post_gen_project.py

```diff
@@ -207,6 +207,24 @@ def handle_js_runner(choice, use_docker, use_async):
         remove_gulp_files()
 
 
+def remove_prettier_pre_commit():
+    with open(".pre-commit-config.yaml", "r") as fd:
+        content = fd.readlines()
+
+    removing = False
+    new_lines = []
+    for line in content:
+        if removing and "- repo:" in line:
+            removing = False
+        if "mirrors-prettier" in line:
+            removing = True
+        if not removing:
+            new_lines.append(line)
+
+    with open(".pre-commit-config.yaml", "w") as fd:
+        fd.writelines(new_lines)
```

Comment:
I've tried to use PyYAML, but I'm struggling to control the formatting of the output. This should hopefully do the job.